src/main/resources/db/migration/V1__revinfo.sql

```diff
@@ -0,0 +1,16 @@
+--  REVTYPE
+--  ADD – 0 - an INSERT SQL statement has created the entity in question
+--  MOD – 1 - an UPDATE SQL statement has modified our entity
+--  DEL – 2 - a DELETE SQL statement has removed the audited entity from the database table
+
+CREATE TABLE revinfo
+(
+    rev      INTEGER GENERATED BY DEFAULT AS IDENTITY ( START WITH 1 ),
+    revtstmp BIGINT,
+    PRIMARY KEY (rev)
+);
+
+CREATE SEQUENCE hibernate_sequence INCREMENT 1 MINVALUE 1
```

Comment:
This sequence is already created in the `V4__base_offer.sql`

Reply:
And maybe we should use here other sequence, since `hibernate_sequence` we are already using for offer ids.

Reply:
OK, I will check that, thanks!

Reply:
@sfc-gh-hlipinski since the JPA inheritance strategy is now changed to `TABLE_PER_CLASS`, isn't this script `V4__base_offer.sql` (adding base_offer) table unnecessary now?

Reply:
I removed the `V4__base_offer.sql` and everything works fine - it wasn't needed finally.
I also renamed subsequent scripts to match the version numbers.

Reply:
And, as far as I read in the docs, hibernate envers doesn't allow us to specify the name of the sequence, so we have to stick with `hibernate_sequence` both for the entity IDs, and for revision table.